### PR TITLE
Addingphyloloss experimental lsf additions

### PR DESCRIPTION
**Pull request by harishB97**

### CODE REVIEW

File: analysis/tsneLSF.py [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+        if cuda is not None:
+            image2 = image2.cuda()
+        image2 = image2.permute(0, 3, 1, 2).to(memory_format=torch.contiguous_format)
+        image2 = ((image2 + 1.0)*127.5).clip(0,255).type(torch.uint8)
+        images = image2 if images is None else torch.cat([images, image2]).detach()
+        fine_labels = fine_label if fine_labels is None else torch.cat([fine_labels, fine_label]).detach()
+        file_names = file_names + file_name
+
+    # Construct the images
+    width = 12000
```
Comment (elhamod) [inline analysis/tsneLSF.py]: This file is still not cleaned up. Also, please if it is mostly same as tsne.py, try to keep one file and just make necessary changes. If the diff is significant enough, it is fine

**Review by elhamod — CHANGES_REQUESTED** [review]

File: analysis/tsneLSF.py (hunk 1 above)
Comment (harishB97) [reply analysis/tsneLSF.py]: I've removed the extra comment lines that were left. 
The difference is not significant, only a few changes, but Mridul has also done some changes from his end on this file for cw, so thought once all clean up is done, we could merge it into one file. Is that fine?

Comment (elhamod) [reply analysis/tsneLSF.py]: It is much more efficient to do it now. because we will have more conflicts later

**Review by elhamod — CHANGES_REQUESTED** [review]

File: analysis/tsneLSF.py (hunk 1 above)
Comment (harishB97) [reply analysis/tsneLSF.py]: Have updated the tsne.py file to be common for all three. Modified loading utils, had to combine the three functions into one. 
A minor change need to be done in the tsne-vqgan.yaml. The `isOriginalVQGAN: true` need to be replaced with `model_name: 'VQGAN'`. tsne-cw.yaml also need to be updated the same way. Everything else should be fine

**Review by elhamod — APPROVED** [review]

**Pull request merged**